### PR TITLE
[10.0][MIG] web_advanced_search_wildcard module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Review by eLBati — APPROVED** [review]
LGTM

File: web_advanced_search_wildcard/static/src/js/search.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "use strict";
+
+    var core = require('web.core');
+    var search_filters = require('web.search_filters');
+
+    var _lt = core._lt;
+
+   search_filters.ExtendedSearchProposition.Char.prototype.operators.push(
+            {value: '=ilike', text: _lt("matches")}
+        );
```
Comment (yajo) [inline web_advanced_search_wildcard/static/src/js/search.js]: Use consistent indentation please.

**Review by yajo — CHANGES_REQUESTED** [review]
Please reduce all translation commits into one: https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-transifex

File: web_advanced_search_wildcard/static/src/js/search.js (hunk 1 above)
Comment (tafaRU) [reply web_advanced_search_wildcard/static/src/js/search.js]: do you mean as follows?

```
search_filters.ExtendedSearchProposition.Char.prototype.operators.push(
    {value: '=ilike', text: _lt("matches")}
);
```
4 spaces are ok for indentation?

Comment (yajo) [reply web_advanced_search_wildcard/static/src/js/search.js]: yes exactly 😃

**Comment by tafaRU** [comment]
@Yajo, all your comments should be addressed. Please check it.

**Comment by yajo** [comment]
Please squash 7fea2e8 also with the translation commits, it's something that shouldn't have happened.

**Comment by tafaRU** [comment]
@Yajo, done.

File: web_advanced_search_wildcard/static/src/js/search.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+odoo.define('web_advanced_search_wildcard', function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var search_filters = require('web.search_filters');
+
+    var _lt = core._lt;
+
+   search_filters.ExtendedSearchProposition.Char.prototype.operators.push(
```
Comment (yajo) [inline web_advanced_search_wildcard/static/src/js/search.js]: This indentation is inconsistent also (sorry to bother you again with this, but you should have fixed it the 1st time 😋)

File: web_advanced_search_wildcard/views/template.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+        <template id="assets_backend" name="web_advanced_search_wildcard assets" inherit_id="web.assets_backend">
+            <xpath expr="." position="inside">
+                <script type="text/javascript" src="/web_advanced_search_wildcard/static/src/js/search.js"/>
+            </xpath>
+        </template>
```
Comment (yajo) [inline web_advanced_search_wildcard/views/template.xml]: please be consistent here too

**Review by yajo — CHANGES_REQUESTED** [review]
I reviewed again and it still seems inconsistent... 😕

File: web_advanced_search_wildcard/static/src/js/search.js (hunk 2 above)
Comment (tafaRU) [reply web_advanced_search_wildcard/static/src/js/search.js]: No problem! Fixed now.

File: web_advanced_search_wildcard/views/template.xml (hunk 3 above)
Comment (tafaRU) [reply web_advanced_search_wildcard/views/template.xml]: done.

**Review by yajo — APPROVED** [review]

File: web_advanced_search_wildcard/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# Copyright 2016 Therp BV <http://therp.nl>.
+# Copyright 2017 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Wildcard in advanced search",
+    "summary": "Webmodule to add wildcard operators in advanced search field",
+    "version": "10.0.1.0.0",
+    "category": "web",
+    "license": "AGPL-3",
+    "author": "initOS GmbH & Co. KG,Odoo Community Association (OCA),Therp BV",
```
Comment (simahawk) [inline web_advanced_search_wildcard/__manifest__.py]: shouldn't OCA stays always at the end? :thinking:

**Review by simahawk — APPROVED** [review]

File: web_advanced_search_wildcard/__manifest__.py (hunk 4 above)
Comment (tafaRU) [reply web_advanced_search_wildcard/__manifest__.py]: @simahawk, just changed!

**Comment by tafaRU** [comment]
@Yajo, I think this could be merged.
Thank you in advance.

**Pull request merged**